README.md

```diff
@@ -1,2 +1,4 @@
 # drafter
 Snowcrash parser harness
+
+[![Circle CI](https://circleci.com/gh/apiaryio/drafter.svg?style=svg&circle-token=e9446ed7fa2c180888e9fde172cf4cd6f57479f4)](https://circleci.com/gh/apiaryio/drafter)
```

Comment:
And this badge should be either on the header line `# drafter` or just after it.

*I prefer on the header line*